components/list/list-item-checkbox-mixin.js

```diff
@@ -147,13 +147,16 @@ export const ListItemCheckboxMixin = superclass => class extends SkeletonMixin(s
 	}
 
 	_renderCheckbox() {
+		const indeterminate = this.selectionInfo.state === SelectionInfo.states.some
```

Comment:
If I understand this correctly, if there are nested list items that were initialized in the `selected` + `disabled` state and then the user clicks select-all, those items would effectively be treated as if they were not selected when determining this item's checkbox state, ya? Are we sure that's the behaviour we want?

Reply:
After reading the code further below in `getSelectionInfo()`, this isn't really the number of disabled keys. Rather, it is the number of keys for items that are simultaneously not `selected` and `disabled`.